### DISCUSSION TITLE
How to remove certs created (received) by ACME plugin

### DISCUSSION THREAD

**Discussion by MarkYQJ** [Kong/kong#9102] [Help] [answered]
Applying ACME plugin, the [document](https://docs.konghq.com/hub/kong-inc/acme/) says manually delete those certificates after testing.
The certs created (or received) by ACME plugin are stored in `config.storage`; in my case (run KongIngressController in DB mode, with postgreSQL DB), it's `kong`; may I know how to delete certificates? just connect to postgreSQL, database `kong`, table `acme_storage`, and remove?

**Comment by hanshuebner** [comment, marked as answer]
Hello @MarkYQJ,

thank you for your question.  At this point, there is no CLI or API functionality to delete certificates, so deleting them using SQL is the way to go.

Kind regards,
Hans
